src/dune_patch_parser/dune_patch_parser.mli

```diff
@@ -0,0 +1,4 @@
+open Stdune
+
+val files_of_patch : dir:Path.t -> string -> Path.t list
+val files_of_patch_file : dir:Path.t -> Path.t -> Path.t list
```

Comment:
Probably you could just move the entire implementation of the patch action here (minus the engine scaffolding) and write even more extensive unit tests.

Reply:
OK I'll call this library `dune_patch` too.